internal/ingress/controller/nginx.go

```diff
@@ -779,9 +787,8 @@ func (n *NGINXController) ConfigureDynamically(pcfg *ingress.Configuration) erro
 		return err
 	}
 
-	glog.V(2).Infof("posting backends configuration: %s", buf)
+	glog.Infof("posting backends configuration: %s", buf)
```

Comment:
This is gonna spam the logs with not so valuable information. It is important to see what's being posted only during debugging.

What do you think we should alway log without escalated logging level?

Reply:
change reverted